plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlClient.java

```diff
@@ -377,6 +387,44 @@ public void testConvertNotExpression()
                 .hasValue("NOT ((\"c_varchar\") IS NOT NULL)");
     }
 
+    @Test(dataProvider = "testTypes")
+    public void testConvertCast(Type testedType)
+    {
+        assertThat(JDBC_CLIENT.convertPredicate(SESSION,
+                translateToConnectorExpression(
+                        new Cast(
+                                new SymbolReference("value_symbol"),
+                                toSqlType(testedType)),
+                        Map.of("value_symbol", testedType)),
+                Map.of("value_symbol", VARCHAR_COLUMN)))
+                .hasValue(format("CAST (\"c_varchar\" AS %s)", JDBC_CLIENT.mapTypeSignature(SESSION, testedType.getTypeId()).get()));
```

Comment:
It looks a bit like testing code that uses `mapTypeSignature` while using `mapTypeSignature` as a reference.

Instead, provide expected remote type name as a test method parameter.